_src/assets/js/partials/localStorage.js

```diff
@@ -88,15 +96,16 @@ function printLocalStorage() {
     let valuePal = saveData.palette - 1;
     let classPal = palettesValue[valuePal];
 
-    function removeClassPal() {
-      for (let i = 0; i < palettesValue.length; i++) {
-        boxCard.classList.remove(palettesValue[i]);
-      }
+
+    for (const value of palettesValue) {
```

Comment:
In this point I think it will be nice add a map function in stead of a for...what do you think? @claraharguindey